articles/native-platforms/android/10-customizing-lock.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Customizing Lock
+description: This tutorial will show you how to modify the Lock activity look and feel.
+seo_alias: android
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+  
+  <%= include('../../_includes/_github', {
+  link: 'https://github.com/inaka/auth0-android-sample/tree/10-Customizing-Lock',
+}) %> 
+
+
+### Before Starting
+
+We will assume that you completed the [Login Tutorial](01-login.md) before starting this one. 
+
+### 1. Create a custom theme
+
+Your first step is to create a `Theme` builder 
+in which you will specify your desired modifications. In this case, we modify the header `Title`:
```

Comment:
Not sure what you mean here, but I guess it should be either:
- the header's `Title`

or

- the `Title` header